cmd/platform/user.go

```diff
@@ -86,26 +89,58 @@ var deleteAllUsersCmd = &cobra.Command{
 	RunE:    deleteAllUsersCommandF,
 }
 
-var migrateAuthCmd = &cobra.Command{
-	Use:   "migrate_auth [from_auth] [to_auth] [match_field]",
-	Short: "Mass migrate user accounts authentication type",
-	Long: `Migrates accounts from one authentication provider to another. For example, you can upgrade your authentication provider from email to ldap.
+var migrateAuthToSamlCmd = &cobra.Command{
```

Comment:
why the change of name here? were you planning to add this as another option?

Reply:
Yes, I have created 2 differente commands, because the previous one is quite different in the expected parameters. I haven't a strong opinion here about how must be the commands, but based on how cobra works, have more sense 2 have 2 different commands which receive differente parameters. In the PR description I include another proposal (based on migrate_auth subcommands). I can merge all together in the old migrateAuth command, but I think is more confusing and difficult to explain the command in the help.

Reply:
Actually I expect some kind of feedback from @jasonblais about this new interface.